### PR TITLE
add notion native redirector tabs

### PR DESCRIPTION
**Pull request by erchn**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
     "autoCloseUrlsConf": {
       "globalprotect": "gpcloudservice.com/SAML20/SP/ACS",
       "zoom": "zoom.us*success",
-      "sdm": "app.strongdm.com/app/auth/complete"
+      "sdm": "app.strongdm.com/app/auth/complete",
+      "notion native": "https://www.notion.so/native"
```
Comment (dalyons) [inline manifest.json]: i think coz i abuse the key as an element id it would have to be `notion_native`

**Review by dalyons — CHANGES_REQUESTED** [review]

File: manifest.json (hunk 1 above)
Comment (erchn) [reply manifest.json]: it seems to work, but I can change if you want.

Comment (erchn) [reply manifest.json]: image

Comment (erchn) [reply manifest.json]: updated

**Review by dalyons — APPROVED** [review]

**Pull request merged**